docs/source/transformers/default-values.rst

```diff
@@ -30,17 +30,17 @@ Providing the value manually has always a priority over the default.
   // Butterfly(5, "Steve", "yellow")
 
 
-Disabling default values in generated transformer
+Enabling default values in generated transformer
```

Comment:
I guess we have to modify the whole page, starting from

> Field's default value is automatically used as a target value when constructing

which no longer holds